### PR TITLE
BOSA21Q1-130 0 suggestions/initiaves > singular in FR

### PR DESCRIPTION
**Pull request by MathieuGillard**
## Description
- What?
- - In FR, "0 suggestion" or "0 pétition" should be singular and not plural. So it needs the same code as for "1 suggestion/1pétition".
- Why?
- - French rules
- How?
- - Override the partial count views for initiative and suggestion.

## Ticket
[BOSA21Q1-130](https://belighted.atlassian.net/browse/BOSA21Q1-130?atlOrigin=eyJpIjoiM2VlOTE1ZjdkNDc1NGZmMGIzNDYzMzFlYTliZmUxZGEiLCJwIjoiaiJ9)

## Type of changes

- [ ]  Docs changes
- [ ]  Refactoring
- [ ]  Dependency upgrade
- [ ]  Bug fix
- [x]  New feature
- [ ]  Breaking changes

### CODE REVIEW

**Review by malbert — APPROVED** [review]

**Review by sergei-krylov — CHANGES_REQUESTED** [review]
- use the existing translations
- fix wrong FR translation in suggestions module
- check the Rails/I18n built in methods or ways to use translation keys based on count, to avoid `if count <= 1 ... else ... ` checks

Please apply the format for commit messages and PRs as `ticket-ID Commit message`, for example `BOSA21Q1-130 Update suggestion translations`

File: app/views/decidim/initiatives/initiatives/_count.html.erb [hunk 1]
```diff
@@ -1 +1 @@
-<%= t(".title", count: initiatives.total_count) %>
+<%= t(".title", count: initiatives.to_a.count) %>
```
Comment (sergei-krylov) [inline app/views/decidim/initiatives/initiatives/_count.html.erb:1]: Why did you change the logic here? why not continue to use the `total_count`?

File: app/views/decidim/suggestions/suggestions/_count.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= t(".title", count: suggestions.to_a.count) %>
```
Comment (sergei-krylov) [inline app/views/decidim/suggestions/suggestions/_count.html.erb:1]: You don't need to override the partial from suggestion module. And same question as above - why logic changes? what's the purpose?

File: config/locales/decidim-initiatives/de.yml [hunk 3]
```diff
@@ -346,6 +346,7 @@ de:
             other: und %{count} weitere Leute
         count:
           title:
+            zero: "%{count} Petition"
```
Comment (sergei-krylov) [inline config/locales/decidim-initiatives/de.yml:349]: This is not correct, read ticket description.

File: config/locales/decidim-initiatives/en.yml [hunk 4]
```diff
@@ -151,6 +151,11 @@ en:
           select_scope_label: Scope
 
       initiatives:
+        count:
+          title:
+            zero: "%{count} initiative"
```
Comment (sergei-krylov) [inline config/locales/decidim-initiatives/en.yml:156]: This is not correct, read ticket description.

File: config/locales/decidim-initiatives/nl.yml [hunk 5]
```diff
@@ -408,6 +408,7 @@ nl:
             other: en nog %{count} mensen
         count:
           title:
+            zero: "%{count} initiatief"
```
Comment (sergei-krylov) [inline config/locales/decidim-initiatives/nl.yml:411]: This is not correct, read ticket description.

File: config/locales/decidim-suggestions/en.yml [hunk 6]
```diff
@@ -6,3 +6,11 @@ en:
         area_ids: Areas
       suggestions_type:
         area_enabled: Enable admins to choose the area for suggestion
+  decidim:
+    suggestions:
+      suggestions:
+        count:
+          title:
+            zero: "%{count} suggestion"
```
Comment (sergei-krylov) [inline config/locales/decidim-suggestions/en.yml:14]: These nodes shouldn't be here, update them in suggestions module.
And this one is not correct, read ticket description.

File: config/locales/decidim-suggestions/fr.yml [hunk 7]
```diff
@@ -6,3 +6,11 @@ fr:
...
       suggestions_type:
         area_enabled: Permettre aux administrateurs de choisir la zone de suggestion
+  decidim:
+    suggestions:
+      suggestions:
+        count:
+          title:
+            zero: "%{count} suggestion"
+            one: "%{count} suggestion"
+            other: "%{count} suggestions"
```
Comment (sergei-krylov) [inline config/locales/decidim-suggestions/fr.yml:16]: These nodes shouldn't be here, update them in suggestions module.

File: config/locales/decidim-suggestions/nl.yml [hunk 8]
```diff
@@ -6,3 +6,11 @@ nl:
         area_ids: Gebieden
       suggestions_type:
         area_enabled: Stel beheerders in staat het gebied voor suggestie te kiezen
+  decidim:
+    suggestions:
+      suggestions:
+        count:
+          title:
+            zero: "%{count} suggestie"
```
Comment (sergei-krylov) [inline config/locales/decidim-suggestions/nl.yml:14]: These nodes shouldn't be here, update them in suggestions module.
And this one is not correct, read ticket description.

**Review by sergei-krylov — CHANGES_REQUESTED** [review]

File: config/locales/decidim-initiatives/en.yml [hunk 9]
```diff
@@ -151,6 +151,10 @@ en:
           select_scope_label: Scope
 
       initiatives:
+        count:
+          title:
+            one: "%{count} initiative"
+            other: "%{count} initiatives"
```
Comment (sergei-krylov) [inline config/locales/decidim-initiatives/en.yml:157]: these are the same as in initial initiatives en.yml from the decidim gem, correct? so we can skip it and don't add these.

File: config/locales/decidim-suggestions/en.yml [hunk 10]
```diff
@@ -5,4 +5,4 @@ en:
       suggestion:
         area_ids: Areas
       suggestions_type:
-        area_enabled: Enable admins to choose the area for suggestion
+        area_enabled: Enable admins to choose the area for suggestion
```
Comment (sergei-krylov) [inline config/locales/decidim-suggestions/en.yml:8]: if we can get rid of these trail newline chars that would be perfect. for this andfr/nl files too, the changes are not related.

**Review by sergei-krylov — APPROVED** [review]

**Comment by sergei-krylov** [comment]
@malbert please squash commits on merge, there was some unnecessary changes in the middle.
